Sources/StreamChat/StateLayer/ConnectedUser.swift

```diff
@@ -154,7 +154,7 @@ public final class ConnectedUser {
     ///
     /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
     ///
-    func loadBlockedUsers() async throws -> [BlockedUserDetails] {
+    public func loadBlockedUsers() async throws -> [BlockedUserDetails] {
```

Comment:
Should we add it to the changelog?

Reply:
Thought about it, but decided not to. However, I don't have strong opinion, I can add it.

Reply:
added